autopilot/contractor.go

```diff
@@ -224,6 +224,13 @@ func (c *contractor) performContractMaintenance(ctx context.Context, w Worker) (
 		return false, fmt.Errorf("failed to run contract checks, err: %v", err)
 	}
 
+	// declare helper to add contracts to the updated set without adding
+	// duplicates.
+	isInSet := make(map[types.FileContractID]struct{})
```

Comment:
can we rename this to `isInUpdatedSet` so we're consistent with `isInCurrentSet`